scipy/stats/tests/test_hypotests.py

```diff
@@ -4,10 +4,10 @@
 from numpy.testing import (assert_, assert_equal, assert_allclose,
                            assert_almost_equal)  # avoid new uses
 
+import pytest
 from pytest import raises as assert_raises
 from scipy.stats._hypotests import (epps_singleton_2samp, cramervonmises,
-                                    _cdf_cvm)
-import scipy.stats as stats
+                                    _cdf_cvm, barnard_exact)
```

Comment:
```suggestion
                                    _cdf_cvm, barnard_exact)
import scipy.stats as stats                                    
```

This seem to be a mistake in the merge as it is used in the `TestSomersD` tests everywhere.

Reply:
Yup, I'm tweaking the `#noqa` line too. Just a minute.